updatecli/updatecli.d/compose.yml

```
@@ -0,0 +1,51 @@
+---
+title: "Bump DockerCompose version"
+sources:
+  dockerComposeVersion:
+    # Version is retrieved from the GitHub source mirror: if there is a release, then you're sure that the artifacts are available
+    kind: githubRelease
+    name: Get the latest Docker Compose version
+    spec:
+      owner: "docker"
+      repository: "compose"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: regex
```

It seems that https://github.com/docker/compose uses semantic versioning so something like seems easier to read

```suggestion
        kind: semver
        pattern: "~1"
```